GUI/QtQuick/QtQuickAdapter.cpp

```diff
@@ -173,6 +173,14 @@ QtQuickAdapter::QtQuickAdapter(const std::shared_ptr<spdlog::logger> &logger)
    {
       setSetting(key, settingsController_->getParam(key));
    });
+
+
+   connect(settingsController_.get(), &SettingsController::reset, this, [this]()
+   {
+      if (settingsController_->hasParam(ApplicationSettings::Setting::SelectedWallet)) {
+         emit requestWalletSelection(settingsController_->getParam(ApplicationSettings::Setting::SelectedWallet).toInt());
```

Comment:
The order of wallets is actually not guaranteed at loading, so the saved index can select a different wallet than was saved previously. Better to use name or wallet id, but let's leave it as is for now.